tests/unit/pubsub_v1/publisher/batch/test_thread.py

```diff
@@ -287,18 +287,43 @@ def test_publish_updating_batch_size():
     assert batch.size > 0  # I do not always trust protobuf.
 
 
-def test_publish_not_will_accept():
-    batch = create_batch(topic="topic_foo", max_messages=0)
-    base_request_size = types.PublishRequest(topic="topic_foo").ByteSize()
+def test_publish():
```

Comment:
Can you create a variant of this where max_messages = 0? We want to ensure that at least one message gets in the batch if it's <= _SERVER_PUBLISH_MAX_BYTES.

Reply:
Pushed